src/Component/login.js

```diff
@@ -0,0 +1,124 @@
+// eslint-disable-next-line import/no-cycle
+import { onNavigate } from '../main.js';
+import { iniciarSesion, usuarioExistente, google } from '../lib/index.js';
+import { modal } from '../Component/modal.js';
+
+export const login = () => {
+  const seccionGeneral = document.createElement('section');
+  const tabla = document.createElement('IMG');
+  tabla.src = './images/login.png';
+  tabla.setAttribute('id', 'cocina');
+  tabla.setAttribute('alt', 'Tabla de cocina');
+  tabla.setAttribute('class', 'desktop');
```

Comment:
consistencia, están medio raros los nombres de las variables, unas en español y otras en ingles
```suggestion
export const login = () => {
  const sectionContent = document.createElement('section');
  const imgTable = document.createElement('IMG');
  imgTable.src = './images/login.png';
  imgTable.setAttribute('id', 'cocina');
  imgTable.setAttribute('alt', 'Tabla de cocina');
  imgTable.setAttribute('class', 'img-table');
```